zio-http/src/main/scala/zio/http/api/Out.scala

```diff
@@ -21,16 +21,26 @@ object Out               {
 
   def value[A](implicit schema: Schema[A]): Out[A] = Value(schema)
 
-  final case class Value[Output](schema: Schema[Output])    extends Out[Output]                           {
+  final case class Value[Output](schema: Schema[Output]) extends Out[Output] {
     type Atom = Output
     type Type = Output
 
     override def bodySchema: Schema[Output] = schema
   }
+
+  final case class AddHeader(key: String, value: String) extends Out[Unit] {
```

Comment:
I discovered there is no need for `Out`. We can do everything we need with `In`. 

Because `In` is bidirectional, it means we can take a value from middleware (output) and turn it into headers.